README.adoc

```diff
@@ -75,12 +75,44 @@ brew tap advancedtelematic/otaconnect
 brew install aktualizr
 ----
 
+You can build and install the latest development version of aktualizr on MacOS (current head of the development branch):
+----
+brew tap advancedtelematic/otaconnect
+brew install --HEAD aktualizr
+----
+
+If any of the previous release versions of aktualizr has been installed before make sure you `unlink` it before
+installing the HEAD version
+
+----
+brew unlink aktualizr
+brew install --HEAD aktualizr
+----
+
+You can switch back to the release version by unlinking and installing again:
+
+----
+brew unlink aktualizr
+brew install aktualizr
+----
+
 You can also build it yourself, with basic dependencies from homebrew. You can install the necessary dependencies as follows:
 
 ----
 brew install asn1c boost cmake libarchive libsodium pkgconfig python3
 ----
 
+and run the following from the aktualizr project directory:
+
+----
+export CXXFLAGS=-w
+cmake -S . -B build -DBoost_USE_MULTITHREADED=ON
+cmake --build build --target all -- -j8
+
+./build/src/aktualizr_primary/aktualizr --version
+----
+
+
 If you also want to compile the SOTA tools:
```

Comment:
Do we still not build the SOTA tools by default for the Mac build? Not worth messing with right now, but I wonder if we should do that for the default homebrew bottle if we don't already.

Reply:
> Do we still not build the SOTA tools by default for the Mac build?
No, I am not sure if we needed it before.

SOTA tools are not included into the build brewed by the aktualizr formula.

Would you like me to remove this instruction from the doc.

Reply:
Nah, it's fine as is for now.